components/Dialogs/src/MDCAlertController.h

```diff
@@ -49,6 +49,9 @@
 /** Alert controllers must be created with alertControllerWithTitle:message: */
 - (nullable instancetype)initWithCoder:(nonnull NSCoder *)aDecoder NS_UNAVAILABLE;
```

Comment:
This doesn't make sense to me - Why can't I archive my view hierarchy with an alertController being shown?